orchestration/dagster_orchestration/hca_orchestration/sensors.py

```diff
@@ -4,15 +4,15 @@
 
 from dagster import sensor, RunRequest, SkipReason
 
-from hca_orchestration.contrib.argo_workflows import ArgoArchivedWorkflowsClientMixin, ExtendedArgoWorkflow
+from hca_orchestration.contrib.argo_workflows import ArgoArchivedWorkflowsClient, ExtendedArgoWorkflow
 from hca_orchestration.resources.base import default_google_access_token
 
 
 # boundary before which we don't care about any workflows in argo
-ARGO_EPOCH = datetime(2021, 3, 1)
+ARGO_EPOCH: datetime = datetime(2021, 3, 1)
```

Comment:
I still think it's worth bumping this.

Reply:
agreed, going to do it in the other pr before merging